### PR TITLE
Explain CLI and Importer update

### PR DESCRIPTION
**Pull request by felixbb**
- Importing more inventory types.
- Added CLI

### CODE REVIEW

File: google/cloud/security/common/data_access/forseti.py [hunk 1]
```diff
@@ -184,17 +186,105 @@ def __repr__(self):
...
+              [GroupMembers, Groups])
     TABLE_CACHE[timestamp] = result
     return result
 
 
 class Importer(object):
     """Forseti data importer to iterate the inventory and policies."""
 
-    DEFAULT_CONNECT_STRING = 'mysql://root@127.0.0.1:3306/forseti_security'
+    DEFAULT_CONNECT_STRING = 'mysql://felix@127.0.0.1:3306/forseti_security'
```
Comment (mirons-google) [inline google/cloud/security/common/data_access/forseti.py]: Is 'felix' intended?

File: google/cloud/security/common/data_access/forseti.py [hunk 2]
```diff
@@ -213,13 +303,71 @@ def _get_latest_snapshot(self):
...
+                self.session.query(folders)
+                .filter(folders.parent_type == 'folder')
+                .filter(folders.parent_id.in_(
+                    [f.folder_id for f in folder_set]))
+                .all()
+                )
+
         for res_type, table in tables:
-            for item in self.session.query(table).all():
+            for item in self.session.query(table).yield_per(1024):
```
Comment (mirons-google) [inline google/cloud/security/common/data_access/forseti.py]: You're using 1024 as a 'magic number' here and other places.. Consider making this a global or a kwarg.

File: google/cloud/security/iam/cli.py [hunk 3]
```diff
@@ -0,0 +1,521 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" IAM Explain CLI. """
+
+#pylint: disable=too-many-locals,missing-type-doc,missing-param-doc
```
Comment (mirons-google) [inline google/cloud/security/iam/cli.py]: You'll need to remove the doc ones as we are requiring proper Google style docs now.

File: google/cloud/security/common/data_access/forseti.py (hunk 1 above)
Comment (felixbb) [inline google/cloud/security/common/data_access/forseti.py]: No, thanks for taking such a close look :)

File: google/cloud/security/common/data_access/forseti.py (hunk 2 above)
Comment (felixbb) [inline google/cloud/security/common/data_access/forseti.py]: created PER_YIELD constant

**Comment by blueandgold** [comment]
Adding `cla: yes` label, as the PR author is a googler at the time of this PR was created.

**Pull request merged**